corehq/apps/app_manager/static/app_manager/js/commcaresettings.js

```diff
@@ -245,12 +246,15 @@ hqDefine('app_manager/js/commcaresettings.js', function () {
                     return setting.visible();
                 });
             });
-            section.reallyCollapse = ko.computed(function () {
-                var el = document.getElementById(section.id);
-                return section.collapse &&
-                    (!el || !el.classList.contains("in")) &&
-                    !_(section.settings).some(function (setting) { return setting.hasError(); });
-            });
+            section.collapse = hqImport("app_manager/js/section_changer.js").shouldCollapse("app-settings", section.id, section.collapse);
+            if (!COMMCAREHQ.toggleEnabled('APP_MANAGER_V2')) {
```

Comment:
This is old accordion logic that forces accordion sections on the app settings page to be open if they have any errors. It's good logic, it isn't used consistently (isn't in forms / modules / vellum) and I had trouble getting this to play nicely with the localStorage preferences combined with the default collapse settings from `commcare-settings-layout.yaml`, decided it wasn't worth spending more time getting it to work right now.